frontend/src/components/utils/NavBar/index.tsx

```diff
@@ -96,45 +136,83 @@ const NavBar = ({ headersData }: Props): ReactElement => {
   const getDrawerChoices = (): ReactElement[] => {
     return headersData.map(({ label, href }) => {
       return (
-        <Link
-          {...{
-            component: RouterLink,
-            to: href,
-            color: 'inherit',
-            style: { textDecoration: 'none' },
-            key: label,
-          }}
-        >
-          <MenuItem>{label}</MenuItem>
-        </Link>
+        <ThemeProvider theme={muiTheme}>
+          <Link
+            {...{
+              component: RouterLink,
+              to: href,
+              color:
+                (location.pathname === '/' && label.includes('Home')) ||
```

Comment:
would probably be best to factor this out into a helper function so it's easier to read